src/main/java/tech/allegro/wildsnake/api/ApiRestController.java

```diff
@@ -0,0 +1,51 @@
+package tech.allegro.wildsnake.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import tech.allegro.wildsnake.product.model.Product;
+import tech.allegro.wildsnake.product.repository.ProductRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Created by Szymon on 06.12.2015.
+ */
+
+@RestController
+@RequestMapping("/api/v1")
+public class ApiRestController {
+
+    private ProductRepository productRepository;
+
+    @Autowired
+    public ApiRestController(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    @RequestMapping(value = "/products", method = RequestMethod.GET)
```

Comment:
Please think about format of your output message (XML, JSON, other) and specify "produces" parameter.